dev/Deployment/DeploymentManager.cpp

```diff
@@ -29,25 +29,30 @@ namespace winrt::Microsoft::Windows::ApplicationModel::WindowsAppSDK::implementa
 
         // Only the Microsoft Publisher Id is supported.
         auto expectedPublisherId{ frameworkPackageInfo.Package(0).packageId.publisherId };
-        FAIL_FAST_HR_IF(E_INVALIDARG, CompareStringOrdinal(expectedPublisherId, -1, WINDOWSAPPRUNTIME_IDENTITY_PUBLISHERID, -1, TRUE) != CSTR_EQUAL);
-
-        // The framework naming scheme consists of a prefix (ex: "Microsoft.WindowsAppSDK.") and a possible
-        // postfix (ex: "-1.0-Release"). We assume all packages of this release match this similar naming scheme
-        // but for their specific identifiers. The framework package is input, so we know its constant identifier,
-        // therefore we can derive the prefix and postfix from the rest of the Name attribute of the Package Family.
-        std::wstring frameworkFamilyName{ frameworkPackageInfo.Package(0).packageFamilyName };
-        auto posFrameworkIdentifier{ frameworkFamilyName.find(WINDOWSAPPRUNTIME_FRAMEWORK_PACKAGE_IDENTIFIER) };
-        const int c_frameworkIdentifierLength{ ARRAYSIZE(WINDOWSAPPRUNTIME_FRAMEWORK_PACKAGE_IDENTIFIER) - 1 };
-        auto posFrameworkDelimeter{ frameworkFamilyName.find(WINDOWSAPPRUNTIME_NAME_DELIMETER) };
-        auto packageFamilyPrefix{ frameworkFamilyName.substr(0, posFrameworkIdentifier)};
-        auto packageFamilyPostfix{ frameworkFamilyName.substr(posFrameworkIdentifier + c_frameworkIdentifierLength, posFrameworkDelimeter - (posFrameworkIdentifier + c_frameworkIdentifierLength))};
+        FAIL_FAST_HR_IF(E_INVALIDARG, CompareStringOrdinal(expectedPublisherId, -1, WINDOWSAPPRUNTIME_PACKAGE_PUBLISHERID, -1, TRUE) != CSTR_EQUAL);
+
+        // The framework naming scheme consists of a prefix (ex: "Microsoft.WindowsAppRuntime.")
+        // and a possible VersionTag (ex: "-1.0-Release"). We assume all packages of this release
```

Comment:
NIT: "ex: 1.0-Preview2"

There is no "Release" versiontag. We use channel names as versiontag (with "" for Stable channel).

And we use dot (not dash) between the name and release (1.0 = major.minor= release, or release version, to be pedantic). So technically the versiontag is "preview2", the release version is "1.0", we use a dash delimiter between them and we use a dot delimiter between the name and the version.

Reply:
Fixed in an earlier change